src/index.js

```diff
@@ -256,6 +256,7 @@ export default renderToString;
 
 export {
 	renderToString as render,
+	renderToString as renderToStaticMarkup,
```

Comment:
Super minor nit: Maybe we can add a little test here that checks if it's exported?

https://github.com/preactjs/preact-render-to-string/blob/master/test/index.js

Reply:
Haha I hoped nobody requests this 😂 I‘ll add one 🙈

Reply:
If the branch wouldn't be in a fork I'd add them myself :see_no_evil: